tools/@aws-cdk/spec2cdk/temporary-schemas/us-east-1/aws-lambda-sourcekmskey.json

```diff
@@ -0,0 +1,453 @@
+{
```

Comment:
Remind me agian when will the official schema be updated? We shouldn't include this here.

Reply:
Have removed it. Thanks for pointing out.